src/main/java/in/projecteka/consentmanager/consent/HipConsentNotificationListener.java

```diff
@@ -50,7 +54,7 @@ public void subscribe() throws ClientError {
                 logger.info("Received notify consent to hip for consent artefact: {}",
                         consentArtefact.getConsentId());
 
-                sendConsentArtefact(consentArtefact);
+                sendConsentArtefactToHIP(consentArtefact);
```

Comment:
This call will break the existing implementation right? if gateway and HIP are not ready when you merge this code?

Reply:
@ganesanarun Yes.

Reply:
How do we avoid that? do you have an idea?

Reply:
@ganesanarun Maybe, can we create a new listener for the same queue without disturbing the HIPConsentNotificationListener?